webapps-backup/src/main/java/io/camunda/webapps/backup/BackupServiceImpl.java

```diff
@@ -49,38 +38,39 @@ public class BackupServiceImpl implements BackupService {
 
   private final List<Prio4Backup> prio4BackupIndices;
 
-  private final OperateProperties operateProperties;
+  private final BackupRepositoryProps backupProps;
 
   private final BackupRepository repository;
 
   private String[][] indexPatternsOrdered;
 
   public BackupServiceImpl(
-      @Qualifier("backupThreadPoolExecutor") final ThreadPoolTaskExecutor threadPoolTaskExecutor,
+      final ThreadPoolTaskExecutor threadPoolTaskExecutor,
       final List<Prio1Backup> prio1BackupIndices,
       final List<Prio2Backup> prio2BackupTemplates,
       final List<Prio3Backup> prio3BackupTemplates,
       final List<Prio4Backup> prio4BackupIndices,
-      final OperateProperties operateProperties,
+      final BackupRepositoryProps operateProperties,
       final BackupRepository repository) {
     this.threadPoolTaskExecutor = threadPoolTaskExecutor;
     this.prio1BackupIndices = prio1BackupIndices;
     this.prio2BackupTemplates = prio2BackupTemplates;
     this.prio3BackupTemplates = prio3BackupTemplates;
     this.prio4BackupIndices = prio4BackupIndices;
-    this.repository = new BackupRepositoryWrapper(repository);
-    this.operateProperties = operateProperties;
+    this.repository = repository;
+    backupProps = operateProperties;
   }
 
   @Override
   public void deleteBackup(final Long backupId) {
     repository.validateRepositoryExists(getRepositoryName());
     final String repositoryName = getRepositoryName();
     final int count = getIndexPatternsOrdered().length;
-    final String version = getCurrentOperateVersion();
```

Comment:
will this version cause problems when we merge them?

Reply:
Good point. And technically yes - I didn't think we needed to fix the issue with the version, but possibly we might, as it would also affect Tasklist :thinking:

Reply:
Let's bring it up in the tech channel, and tag Marcos and Svetlana/Thorben

Reply:
I'm working on the merge now and I noticed that tasklist is doing the same actually in its implementations.